provider/helpers.go

```diff
@@ -20,9 +20,9 @@ import (
 	"github.com/hashicorp/terraform-plugin-framework/types"
 )
 
-// shouldPatchInt changed checks if the change between the given state and plan requires sending a
+// ShouldPatchInt changed checks if the change between the given state and plan requires sending a
 // patch request to the server. If it does it returns the value to add to the patch.
-func shouldPatchInt(state, plan types.Int64) (value int64, ok bool) {
+func ShouldPatchInt(state, plan types.Int64) (value int64, ok bool) {
```

Comment:
Why did you change it, its used only from this package

Reply:
I tried to create a new package but removed it and didn't change it back to be in the original name